WebContent/index.jsp

```diff
@@ -1,29 +1,34 @@
 <%@ page language="java" contentType="text/html; charset=ISO-8859-1"
-    pageEncoding="ISO-8859-1"%>
+	pageEncoding="ISO-8859-1"%>
 <%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
 <%@ page import="com.sundar.studentmanagement.student.vo.*"%>
-    
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+
+<!DOCTYPE html>
 <html>
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
 <title>Student Management</title>
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<link href="./plugins/bootstrap.css" rel="stylesheet" />
-<script type="text/javascript" src="./plugins/bootstrap.min.js"></script>

```

Comment:
**bootstrap.min.js** --has to imported after jquery

- because bootstrap has to know, what is **$** means.
- if you didn't import jquery before bootstrap, it will error out **$** is undefined

check the latest code in /jsp/common/allImports.jsp